cmd/osv-scanner/internal/testcmd/run.go

```diff
@@ -20,3 +23,36 @@ func Run(t *testing.T, tc Case) (string, string) {
 
 	return stdout.String(), stderr.String()
 }
+
+func Run(t *testing.T, tc Case) {
```

Comment:
should we expand this name to try make it more assert-y, e.g. `testcmd.RunAndMatchSnapshots`?

Reply:
Yep, I like more descriptive names.